ansible/roles/csm.ncn.move_first_master/tasks/main.yml

```diff
@@ -88,27 +88,26 @@
     new_first_master_hostname: "{{ item }}"
   with_random_choice: "{{ master_ncn_names.stdout.split(',') | reject('search', target_hostname) | list }}"
 
-- name: Get certificate key
+- name: Copy promote master script to new first master
   when:
     - first_master_hostname == target_hostname
   delegate_to: "{{ new_first_master_hostname }}"
-  shell: kubeadm init phase upload-certs --upload-certs 2>&1 | tail -1
-  register: certificate_key
+  template:
+    src: "templates/promote-first-master.sh.j2"
+    dest: "/tmp/promote-first-master.sh"
+    mode: 0755
 
-- name: Build kubeadm config builder script
+- name: Run promote first master script
   when:
     - first_master_hostname == target_hostname
   delegate_to: "{{ new_first_master_hostname }}"
-  template:
-    src: "templates/build-kubeadm.sh.j2"
-    dest: "/tmp/build-kubeadm.sh"
-    mode: 0755
+  shell: /tmp/build-kubeadm.sh
```

Comment:
I believe this should be '/tmp/promote-first-master.sh', not '/tmp/build-kubeadm.sh'?